suprb/optimizer/rule/mutation.py

```diff
@@ -31,6 +34,9 @@ def __call__(self, rule: Rule, random_state: RandomState) -> Rule:
     def execute(self, rule: Rule, random_state: RandomState):
         pass
 
+    def adapt(self, elitist_fitness: float):
```

Comment:
why do we want this for all RuleMutations?

Reply:
removed